mkosi

```diff
@@ -1054,8 +1054,27 @@ def invoke_dnf(args: CommandLineArguments, workspace: str, repositories: List[st
     with mount_api_vfs(args, workspace):
         run(cmdline, check=True)
 
+    if cleanup:
```

Comment:
hmm, this is quite generically named, no? maybe call this "package_manager_cleanup"?

Reply:
It *was* supposed to be generic. My idea was to stuff additional cleanup steps there in the future. E.g. logs or something.